data/Richards_2021/metadata.yml

```diff
@@ -0,0 +1,533 @@
+source:
+  primary:
+    key: Richards_2021
+    bibtype: Article
+    year: 2021
+    author: Jeannine H. Richards, Jonathan J. Henn, Quinn M. Sorenson, Mark A. Adams,
+      Duncan D. Smith, Katherine A. McCulloh and Thomas J. Givnish
+    title: Mistletoes and their eucalypt hosts differ in the response of leaf functional
+      traits to climatic moisture supply
+    journal: Oecologia
+    volume: 195
+    pages: 759--771
+    doi: 10.1007/s00442-021-04867-1
+contributors:
+  data_collectors:
+  - last_name: Richards
+    given_name: Jeannine
+    ORCID: 0000-0002-1736-2126
+    affiliation: Nelson Institute for Environmental Studies, University of Wisconsin-Madison,
+      Madison, WI, USA
+    additional_role: contact
+  austraits_curators: Sophie Yang
+dataset:
+  data_is_long_format: no
+  custom_R_code:                         '
+    data %>% 
+      mutate(
+        date = paste(day, month, year, sep = "-") %>% dmy(),
+        lma = 1/(sla/10000),
```

Comment:
It is fine to leave this, but in general you don't have to convert LMA into SLAto map it in. As long as the units are correct for SLA the values are successfully flipped through unit conversions